### PR TITLE
remove user proposals from public proposals list

### LINKED ISSUE #156: filter proposals in list proposals page
filter proposals  properly in list proposals page. for example `my proposals` shoudn't be shown in `list of proposals`
ll

### PR DESCRIPTION
**Pull request by ChillarAnand**
closes #156

### CODE REVIEW

**Comment by sivaa** [comment]
@kracekumar Can you please review this one?

File: junction/proposals/views.py [hunk 1]
```diff
@@ -71,7 +71,11 @@ def list_proposals(request, conference_slug):
         proposals_qs = proposals_qs.filter(proposal_type__id__in=proposal_type_filter)
         is_filtered = True
 
-    public_proposals_list = proposals_qs.filter(~Q(review_status=PROPOSAL_REVIEW_STATUS_SELECTED))
+    if request.user.is_authenticated():
+        public_proposals_list = proposals_qs.filter(
+            ~Q(review_status=PROPOSAL_REVIEW_STATUS_SELECTED) | ~Q(author=request.user))
```
Comment (kracekumar) [inline junction/proposals/views.py]: How about using exclude ?

**Comment by ChillarAnand** [comment]
closes #136  #166

**Comment by kracekumar** [comment]
Can you rebase ?

**Comment by ChillarAnand** [comment]
@kracekumar done

**Pull request merged**